events/events.go

```diff
@@ -22,3 +22,7 @@ type publisherFunc func(ctx context.Context, topic string, event Event) error
 func (fn publisherFunc) Publish(ctx context.Context, topic string, event Event) error {
 	return fn(ctx, topic, event)
 }
+
+type Subscriber interface {
```

Comment:
What is this interface used for?

Reply:
It declares the `Subscriber` interface and was missing from above.

Reply:
Just meant, nothing is referring to the interface :)

Reply:
I think it is fine here to declare the interface since it should be common across multiple users of the client.